assets/tuned/06-ds-tuned.yaml

```diff
@@ -17,7 +17,11 @@ spec:
       serviceAccountName: tuned
       automountServiceAccountToken: true
       containers:
-      - command: ["/var/lib/tuned/bin/run","start"]
+      - command: ["/bin/sh", "-c"]
+        args:
+          - ln -s /boot/grub2/grub.cfg /etc/grub2.cfg;
+            ln -s /boot/grub2/grub.cfg /etc/grub2-efi.cfg;
```

Comment:
Are both `grub2.cfg` and `grub2-efi.cfg` needed?

Reply:
I needed to create the symlinks because tuned reads those grub2.cfg and grub2-efi.cfg files instead of reading them from /boot/grub2, that is the original source. So yes, they are needed. In my case i tested with grub2.cfg (bios boot), but i guess grub2-efi.cfg is used for UEFI.